src/components/shared/form_inputs/useInputFocusing.ts

```diff
@@ -0,0 +1,24 @@
+import { ref, Ref } from 'vue';
+
+type ReturnType = {
+	labelRef: Ref<HTMLLabelElement>,
+	inputRef: Ref<HTMLInputElement>,
+	focus(): void,
+	click(): void,
+};
+
+export function useInputFocusing(): ReturnType {
```

Comment:
Could you add a docblock here to explain what this does? It's like a polyfill that unifies label click behavior across browsers, right?